### PR TITLE
Adds "Enumerate RDP Connection History" Procedure

### PR DESCRIPTION
**Pull request by w0rk3r**

### CODE REVIEW

**Comment by w0rk3r** [comment]
Ref: https://github.com/3gstudent/List-RDP-Connections-History

File: discovery/d03901f3-8061-4c55-880b-034886788e68.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            {
+        	    continue
+            }
+          foreach($Name in $QueryPath)
+            {   
+        	  Try  
+            {
+            	$User = (Get-ItemProperty -Path $RegPath$Name -ErrorAction Stop).UsernameHint
+            	Write-Host "Server:"$Name
+            	Write-Host "User:"$User
```
Comment (khyberspache) [inline discovery/d03901f3-8061-4c55-880b-034886788e68.yml:35]: We can probably set this up for future chaining if we somehow write it in JSON form here

**Review by khyberspache — CHANGES_REQUESTED** [review]

File: discovery/d03901f3-8061-4c55-880b-034886788e68.yml (hunk 1 above)
Comment (w0rk3r) [reply discovery/d03901f3-8061-4c55-880b-034886788e68.yml:35]: Going to take a look tomorrow

Comment (w0rk3r) [reply discovery/d03901f3-8061-4c55-880b-034886788e68.yml]: image

Done

**Review by khyberspache — APPROVED** [review]

**Pull request merged**